assets/inference/environments/minimal-ubuntu22.04-py311-cpu-app-quickstart/asset.yaml

```diff
@@ -0,0 +1,7 @@
+name: minimal-py311-inference-quickstart
```

Comment:
why not keep consistant with the name?

Reply:
minimal-ubuntu22.04-py311-cpu-app-quickstart

Reply:
![image](https://github.com/Azure/azureml-assets/assets/40655746/90805ba5-b1ca-4c4c-951e-eb20acbc9329)
This is what they did for other environments under the same folder. So I tried to follow the pattern.

Reply:
Oh, my bad. Looked at the wrong files. I will change it.